custom/icds_reports/ucr/data_sources/it_report_follow_issue.json

```diff
@@ -49,6 +49,20 @@
         "transform": {},
         "type": "expression"
       },
+      {
+        "display_name": "Supervisor ID",
+        "datatype": "string",
+        "type": "expression",
+        "transform": {},
+        "column_id": "supervisor_id",
+        "expression": {
+          "location_id_expression": {
+            "type": "named",
+            "name": "user_location_id"
+          },
+          "type": "location_parent_id"
```

Comment:
Are hardarware cases owner by supervisor or awc_id. Depending on that this may or may not be correct.

Reply:
yea this is one that's not marked for distribution in the sheet since the owner can be from levels higher than supervisor